forge-gui/res/cardsfolder/rebalanced/a-blessed_hippogriff_tyrs_blesing.txt

```diff
@@ -0,0 +1,18 @@
+Name:A-Blessed Hippogriff
+ManaCost:3 W
+Types:Creature Hippogriff
+PT:2/3
+K:Flying
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ Whenever CARDNAME attacks, target attacking creature without flying gains flying until end of turn.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.attacking+withoutFlying | TgtPrompt$ Select target attacking creature without flying | KW$ Flying
+AlternateMode:Adventure
+Oracle:Flying\nWhenever Blessed Hippogriff attacks, target attacking creature without flying gains flying until end of turn. 
+
+ALTERNATE
+
+Name:A-Tyr's Blesing
+ManaCost:1 W
+Types:Instant Adventure
+A:SP$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target to make indestructable | KW$ Indestructible | SpellDescription$ Target creature gains indestructible until end of turn.
```

Comment:
please remove this TgtPrompt and all of line 17
[also from the non `A-` version]

Forge will automatically parse TgtPromt and this is better than this with the misspelling and all